types/react-dom/test-utils/index.d.ts

```diff
@@ -257,18 +253,18 @@ export function findRenderedDOMComponentWithTag(
 /**
  * Finds all instances of components with type equal to `componentClass`.
  */
-export function scryRenderedComponentsWithType<T extends Component, C extends ComponentClass>(
+export function scryRenderedComponentsWithType<T extends Component>(
     root: Component<any>,
-    type: ClassType<any, T, C>): T[];
+    type: ClassType<any, T, ComponentClass>): T[];
 
 /**
  * Same as `scryRenderedComponentsWithType()` but expects there to be one result
  * and returns that one result, or throws exception if there is any other
  * number of matches besides one.
  */
-export function findRenderedComponentWithType<T extends Component, C extends ComponentClass>(
+export function findRenderedComponentWithType<T extends Component>(
     root: Component<any>,
-    type: ClassType<any, T, C>): T;
```

Comment:
What are the changes in this file for?

Reply:
To fix dtslint issues ;) The type parameter is only used once so the linter throws a stone at you.

Reply:
+1 for "throws a stone"